### ISSUE TITLE
Add chrome installation description

### ISSUE THREAD

**Issue by kAlz** [Mortal/bbdyn#10]
Chrome with Tampermonkey and the script seem to work perfectly fine.

**Comment by mdbech** [comment]
Yeah it does, optionally see attached picture. Therefore this issue can be closed.
![screen shot 2016-04-27 at 12 31 24](https://cloud.githubusercontent.com/assets/13867681/14849601/349a8f5c-0c74-11e6-84e8-9f859c8e463a.png)
